cob_obstacle_distance/include/cob_obstacle_distance/link_to_collision.hpp

```diff
@@ -71,17 +71,17 @@ class LinkToCollision
         urdf::Model model_;
         bool success_;
         std::string root_frame_id_;
-        std::unordered_map<std::string, std::vector<std::string> > self_collision_frames_; /// first: key of part to be checked with, second ignore links
+        std::unordered_map<std::string, std::vector<std::string> > self_collision_map_; /// first: link to be considered 'obstacle', second: links of component to be considered for self-collision checking
```

Comment:
The name of the type (here map) should not be included in the variable name (Clean Code, R. C. Martin) ;-)